packages/blue-sdk/src/chain.ts

```diff
@@ -103,5 +108,40 @@ export namespace ChainUtils {
       explorerUrl: "https://explorer.inkonchain.com",
       identifier: "ink",
     },
+    [ChainId.Unichain]: {
+      name: "Unichain",
+      id: ChainId.Unichain,
+      nativeCurrency: { name: "Ether", symbol: "ETH", decimals: 18 },
+      explorerUrl: "https://uniscan.xyz",
+      identifier: "unichain",
+    },
+    [ChainId.SonicMainnet]: {
+      name: "Sonic",
+      id: ChainId.SonicMainnet,
+      nativeCurrency: { name: "Sonic", symbol: "S", decimals: 18 },
+      explorerUrl: "https://explorer.soniclabs.com",
```

Comment:
```suggestion
      explorerUrl: "https://sonicscan.org/",
```
That's the explorer provided in the doc

Reply:
https://docs.soniclabs.com/sonic/build-on-sonic/getting-started

Reply:
both work but soniscan seems to be more etherscan-like

Reply:
Thanks! the other explorer is shitty